.github/labeler.yml

```
@@ -0,0 +1,14 @@
+🧩 cli: packages/cli/**
+🧩 core: packages/core/**
+🧩 models: packages/models/**
+🧩 nx-plugin: packages/nx-plugin/**
+🧩 eslint-plugin: packages/plugin-eslint/**
+🧩 lighthouse-plugin: packages/plugin-lighthouse/**
+🧩 utils: packages/utils/**
+
+🔬 testing:
+  - packages/**/*.spec.ts
+  - e2e/**
+  - testing-utils/**
+
+🦾 CI/CD: .github/**
```

To distuinguish from issue templates, for example:

```suggestion
🦾 CI/CD: .github/workflows/**
```